### PR TITLE
adding bootstrap theme to app

### PR DESCRIPTION
**Pull request by zak-schlemmer**

### CODE REVIEW

**Comment by zak-schlemmer** [comment]
I have noticed an issue with the home link, covers the prompt for login if not already auth'd.
I'll fix that up to add to this PR later today.

File: app/index.html [hunk 1]
```diff
@@ -17,12 +17,24 @@
     <!-- App Settings and DrupalGap Contrib/Custom Modules -->
     <script type="text/javascript" charset="utf-8" src="settings.js"></script>
     <script src="modules/custom/growmobot/growmobot.js"></script>
-    <script src="themes/growmobot/growmobot.js"></script>
+    <script src="modules/contrib/dg_bootstrap-8.x-1.x/dg_bootstrap.min.js"></script>
```
Comment (signalpoint) [inline app/index.html]: Did you use `./dg dl dg_bootstrap` here? It looks like you did a manual download, the folder name should only be `dg_bootstrap`.

Comment (zak-schlemmer) [reply app/index.html]: I did do it manually.
I did not realize that the dg tool supported this.
Good to know!

Comment (signalpoint) [reply app/index.html]: Check out the `Update the CLI` section on this page to pull in the latest, unless you're still running your fork of DG8, then pull down from upstream.

http://docs.drupalgap.org/8/Developer_Guide/CLI_-_Command_Line_Interface

Comment (zak-schlemmer) [reply app/index.html]: It might be worth adding that here:
https://drupalgap.org/project/dg_bootstrap
That was my reference for this implementation.

Comment (signalpoint) [reply app/index.html]: Each of the project pages on drupalgap.org actually just pull from the README.md file hosted on GitHub for each project, that way the docs for each module can just live in one place, and aggregate around.

Comment (zak-schlemmer) [reply app/index.html]: Cool, that sounds all as it should be.
I'll clean that up / try that out when I get back to fixing the above mentioned issue later today.

**Comment by zak-schlemmer** [comment]
It should be pretty cleaned up now; if you see no issues with the last commit, feel free to merge.

**Pull request merged**